bnelson777/id.ly
# IL-118: Prevent user from decrypting a message intended for someone else

To test:
1. On your main device, create a new card. Share it to your second device.
2. On your second device, send a message to the new card you just scanned and email to yourself. 
3. Decrypt that message on the main device. Message should successfully decrypt. 
4. On main device, clear all data and reload the app. Now try to load the message. You should get a popup saying you can't decrypt a message meant for someone else.
